Installing-Metacontroller-s390x.md

```diff
@@ -0,0 +1,73 @@
+# Installing Metacontroller on an s390x k8s cluster
+
+[Metacontroller](https://github.com/metacontroller/metacontroller) is not available for the s390x platform out of the box. This [PR](https://github.com/metacontroller/metacontroller/pull/771) adds support but in the meantime it is possible to deploy metacontroller on s390x using the following recipe:
+
+## Build Metacontroller
+
+Prequisites:
+
+- an s390x virtual server (e.g. a LinuxONE box)
+- installation of [go](https://go.dev/doc/install)
+- installation of [docker](https://docs.docker.com/engine/install/ubuntu/)
+
+Steps:
+
+- clone the metacontroller repository in the desired version e.g.
+
+    ```bash
+    git clone https://github.com/metacontroller/metacontroller -b v4.7.10
+    ```
+
+- build the metacontroller binary
+
+    ```bash
+    cd metacontroller
+    CGO_ENABLED=0 GOOS=linux GOARCH=s390x /usr/local/go/bin/go build ./pkg/cmd/metacontroller
+    ```
+
+- build the docker image
+
+    ```bash
+    docker build . -t metacontrollerio/metacontroller:v4.7.10
```

Comment:
this should also have the `--platform=linux/s390x` flag to make sure it's built for s390x.